### PR TITLE
[Plugin] opecvmini `rectangle` and `cvtColor`

### PR DESCRIPTION
**Pull request by dannypsnl**
1. `rectangle`[^1] can draw rectangle on given picture buffer
   - [x] customize color
   - [x] thickness = 1
   - [x] lineType = LINE_8
   - [x] shift = 0
2. `cvt_color`[^2] takes _color conversion code_ and _destination channel N_ to transform given matrix

[^1]: https://docs.opencv.org/4.x/d6/d6e/group__imgproc__draw.html#gac865734d137287c0afb7682ff7b3db23
[^2]: https://docs.opencv.org/3.4/d8/d01/group__imgproc__color__conversions.html#ga397ae87e1288a81d2363b61574eb8cab

### CODE REVIEW

**Comment by juntao** [comment]
Hello, I am a [code review bot](https://github.com/flows-network/github-pr-summary/) on [flows.network](https://flows.network/).

It could take a few minutes for me to analyze this PR. Relax, grab a cup of coffee and check back later. Thanks!

File: plugins/wasmedge_opencvmini/opencvmini_func.cpp [hunk 1]
```diff
@@ -147,5 +147,35 @@ WasmEdgeOpenCVMiniBilinearSampling::body(const Runtime::CallingFrame &,
...
+Expect<void> WasmEdgeOpenCVMiniRectangle::body(
+    const Runtime::CallingFrame &, uint32_t SrcMatKey, uint32_t top,
+    uint32_t left, uint32_t bot, uint32_t right, double R, double G, double B,
+    int thickness, int lineType, int shift) {
+  auto Src = Env.getMat(SrcMatKey);
+  if (!Src) {
+    return Unexpect(ErrCode::Value::HostFuncError);
+  }
+
+  cv::Point top_left(top, left);
```
Comment (hydai) [inline plugins/wasmedge_opencvmini/opencvmini_func.cpp:159]: Wrong naming style.

**Review by hydai — CHANGES_REQUESTED** [review]

**Comment by 1awesomeJ** [comment]
Lots of amazing features on this repository:
1. Juntao giving detailed review- Thank you so much for the initiative. It's super helpful for us beginners. Thank you for coming up with it.
2. codecov bot- helping us easily see test coverage
3. Merging getting block when there are pending requested changes.
It's all awesome.

I'm an aspiring LFX mentee on the project "add matrix operations for opencvmini", and these will be quite helpful in helping get started with making contributions.

**Comment by dannypsnl** [comment]
@1awesomeJ Please don’t make irrelevant comment in PR, don’t apologize, I will hide your and this comment. You can create discussion for this kind of content.

**Review by ibmibmibm — APPROVED** [review]

**Review by hydai — APPROVED** [review]
LGTM

**Pull request merged**